docs/generators/csharp.md

```diff
@@ -49,6 +49,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |useCollection|Deserialize array types to Collection&lt;T&gt; instead of List&lt;T&gt;.| |false|
 |useDateTimeForDate|Use DateTime to model date properties even if DateOnly supported. (.net 6.0+ only)| |false|
 |useDateTimeOffset|Use DateTimeOffset to model date-time properties| |false|
+|useLegacyOperationParameterSorting|Preserve legacy operation parameter sorting to avoid a breaking change.| |true|
```

Comment:
can we rename this option to operationParameterSorting, which supports `legacy` and `alphabetical`?

and later we have the option to add another one called `original` if users want the order in the spec.

Reply:
done